fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
I think this skipping is because with this patch @awoods is deleting links to non-existent resources. But for other resources created before the patch this recursive deletion did not take place, so we skip to allow access to those resources. Otherwise we could run the inbound link deletion on a GET, but that seems wrong for some reason.